src/assets/css/styles.css

```diff
@@ -59,7 +59,7 @@ nav ul li a:active, nav ul li a:focus {
 
 /* Hero Section*/
 #hero {
-    height: 100vh;
+    /* height: 100vh; */

```

Comment:
This `100vh` property didn't work because you added the vision content outside of the hero section. But I guess there are other ways to work around it.

Reply:
Yes, I added the vision section outside of hero section. So, are you suggesting that I should add it in the hero section itself?